pages/data-providers/[data-provider-id]/compliance.css

```diff
@@ -15,9 +15,18 @@
   }
 }
 
-.compliance-level {
+.data-overview {
   flex: 2 2 60%;
   width: 100%;
+
+  & > .numbers {
```

Comment:
There is no reason to use nested class. It performs worse. It matters even if we talk about such simple one because we have a lot of selectors.